MAINTAINERS.md

```diff
@@ -26,6 +26,7 @@ Maintainers are assigned the following scopes in this repository:
 | dbluhm          | Daniel Bluhm     |            |      |            |       | Indicio PBC         |
 | dhh1128         | Daniel Hardman   | Maintainer |      |            |       | Provenant           |
 | ianco           | Ian Costanzo     |            |      |            |       | Anonymous Systems   |
+| jamshale        | Jamie Hale       | Maintainer |      |            |       |                     |
```

Comment:
Please add BC Gov as the organization.  Thanks!

Reply:
Updated. I rebased into one commit.